src/Pages/Mypage/Components/Funded/Funded.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import Progressbar from '../../../../Components/Progressbar/Progressbar';
+import './Funded.scss';
+
+class Funded extends Component {
+  render() {
+    const { img, percent, price, title, date, company, catagory } = this.props;
+    return (
+      <div className="funded">
+        <div className="imgBox">
+          <span className="label">리워드</span>
+          <img src={img} alt={catagory} />
+          <div className="test">
+            <Progressbar percent={percent} />
+          </div>
+          {/* <div className="info"> */}
+          <div className="leftText">
+            <span className="percent">{percent}%</span>
+            <span className="price">{price}원 달성</span>
+            {/* <span className="date">{date}일 남음</span> */}
+            {date[0] === '-' ? (
+              <span className="date">종료</span>
+            ) : (
+              <span className="date">{date}일 남음</span>
+            )}
```

Comment:
```suggestion
              <span className="date">{date[0] === '-' ? "종료" : {date}일 남음}</span>
```
content만 바뀔 경우 content에 조건문을 적용해주면 좋겠네요!

Reply:
네 수정했씁니다!